Example/Tests/GINIVisionDocumentTests.swift

```diff
@@ -11,16 +11,18 @@ import XCTest
 
 class GINIVisionDocumentTests: XCTestCase {
     
+    let giniConfiguration: GiniConfiguration = GiniConfiguration()
     var filePickerManager: DocumentPickerCoordinator {
         return DocumentPickerCoordinator()
     }
     
     func testExcedeedMaxFileSize() {
         let higherThan10MBData = generateFakeData(megaBytes: 12)
         
-        let fakeDocument = GiniPDFDocument(data: higherThan10MBData)
+        let pdfDocument = GiniPDFDocument(data: higherThan10MBData)
         
-        XCTAssertThrowsError(try fakeDocument.validate(),
+        XCTAssertThrowsError(try GiniVisionDocumentValidator.validate(pdfDocument,
```

Comment:
I'm not really sure this is the right place for these tests. You moved the validation to `GiniVisionDocumentValidator`. So there's nothing about validation in the document class anymore. If the class doesn't have validation, then its tests shouldn't test the validation.

Just move these tests to the `GiniVisionDocumentValidatorTests` class.